test/util/enketo.js

```diff
@@ -44,7 +44,7 @@ const reset = () => {
 const request = () => {
   global.enketo.callCount += 1;
   const options = { ...global.enketo };
-  Object.assign(global.enketo, without(['callCount'], defaults));
+  Object.assign(global.enketo, without(['callCount', 'state'], defaults));
```

Comment:
There's probably a better way to deal with this enketo error state and maintaining it for two requests in a row.

Reply:
I do think it makes it slightly harder to reason about the mock if some properties are automatically reset and others aren't. Also, it looked like there was one test where you didn't want `wait` to get reset. What about adding a new property to `global.enketo` named something like `autoReset` where if `autoReset` is `true`, this line is called, but if it's set to `false`, then nothing is reset (not `state`, not `wait`)? `autoReset` would then itself get reset after every test (when the `reset()` function is called). Just an idea!

Reply:
Tried out this `autoReset` approach and it seems useful.